features/HHH_CFE-IntTests-client-involvement-type/hhh-cit-2.feature

```diff
@@ -0,0 +1,34 @@
+Feature:
+  " NON-PASSPORTED
+    Defendant x 2
+    Above all upper thresholds
+    Ineligible "
+
+  Scenario: Test that the correct output is produced for the following set of data.
+    Given I am undertaking a certificated assessment
+    And A submission date of "2022-01-24"
+    And I add the following proceeding types in the current assessment:
+      | ccms_code | client_involvement_type |
+      | DA001     | A                       |
```

Comment:
Spreadsheet say involvement type is `D` but gives CCMS code ad `DA001`.

Not sure which is accurate.